packages/config/index.d.ts

```diff
@@ -90,6 +90,7 @@ export interface StackableInterface {
   start: (options: StartOptions) => Promise<void>
   stop: () => Promise<void>
   getUrl: () => string
+  updateContext: (context: Partial<StackableContext>) => Promise<void>
```

Comment:
I would make this optional (here and in the file that invokes it).
Not all stackables might be interested on updating the context.

In general, I think this should be the approach we use for most of the Stackable APIs, excluding some obvious exceptions like `start` and similar.

Reply:
I aggree.  I decided it would be better to have one place with a default stackable methods implementations, than add `stackable?.foo() || null` everywhere. PTAL

Reply:
Once you implement Matteo's modification suggestion below, I'm good! ^^